packages/sources/tp/src/endpoint/price.ts

```diff
@@ -47,14 +48,14 @@ export const generateInputParams = (
 const tpOptions: GeneratePriceOptions = {
   sourceName: 'tpSource',
   streamName: 'TP',
-}
+} as const
 
 const inputParameters = generateInputParams(tpOptions)
 const wsTransport = generateTransport(tpOptions)
 
-export const priceEndpoint = new ForexPriceEndpoint({
```

Comment:
What's the reason of this change ?

Reply:
So right now we have the `price` and `forex` aliases with the `ForexPriceEndpoint` class being the same thing as `PriceEndpoint`. Since we want to add new `commodities` alias as well, I thought it would make more sense to use the base `PriceEndpoint` class rather than continue to use the `ForexPriceEndpoint`.